packages/gestalt/src/Button.js

```diff
@@ -143,6 +155,85 @@ const ButtonWithForwardRef: React$AbstractComponent<
   );
 
   /* eslint-disable react/button-has-type */
+  if (props.type === 'link') {
+    const { href, rel, target } = props;
+
+    return (
+      <Link
+        hoverStyle="none"
+        href={href}
+        ref={innerRef}
+        rel={rel}
+        target={target}
+        inline
```

Comment:
@AlbertCarreras curious to know what happens for non-inline buttons. Should this be conditional based on the passed in props?

Reply:
WIP. The finishings were not done.

Reply:
DONE